ui/app/services/auth.js

```diff
@@ -498,10 +498,11 @@ export default Service.extend({
       authMountPath,
       entityId: authData?.entity_id,
       expireTime: authData?.expire_time,
-      token: authData?.client_token,
+      token: token || authData?.client_token,
       renewable: authData?.renewable,
-      ttl: authData?.lease_duration,
+      ttl: ttl || authData?.lease_duration,
       policies: authData?.policies,
+      mfaRequirement: authData?.mfa_requirement,
```

Comment:
I guess the major difference is previously I spread the auth response in the normalize function (at the top) and then overrode keys with the normalized names. 

Mostly because I was concerned there could be additional data returned by the API's login response that I missed during the refactor. For example, there's a `num_uses` key - I don't think it was used anywhere but wasn't sure if there just wasn't test coverage for it.

Reply:
> The one issue I see with doing that now is that these properties that are explicitly defined are in camelCase and the remaining properties from the spread would now be snake_case which could be confusing.

☝️ yes exactly! I think that would be confusing.

Yep - sounds good! Thanks for thinking that through with me!